plugins/modules/network/routeros/routeros_api.py

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Nikolay Dachev <nikolay@dachev.info>
+# GNU General Public License v3.0+ https://www.gnu.org/licenses/gpl-3.0.txt
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: routeros_api
+version_added: 1.1.0
+author: Nikolay Dachev (@nikolaydachev)
+short_description: Ansible module for RouterOS API
+description:
+  - Ansible module for RouterOS API with python librouteros.
+     This module can add, remove, update, query and execute arbitrary command
+     in routeros via api
+options:
+  hostname:
+    description:
+      - RouterOS hostname api
+    required: true
+    type: str
+  username:
+    description:
+      - RouterOS login user
+    required: true
+    type: str
+  password:
+    description:
+      - RouterOS user password
+    required: true
+    type: str
+  ssl:
+    description:
+      - If is set ssl will be used for RouterOS api connection
+    required: false
+    type: bool
+  port:
+    description:
+      - RouterOS api port. If ssl is set, port will apply to ssl connection.
+        http api - 8728
+        https api - 8729
+    type: int
+  path:
+    description:
+      - Main path for all other arguments.
+        If other arguments are not set, api will return all items in
+         selected path.
+        Example "ip address".
+         Eqvivalent of RouterOS cli "/ip address print"
+    required: true
+    type: str
+  add:
+    description:
+      - Will add selected arguments in selected path to RouterOS config.
+        Example "address=1.1.1.1/32 interface=ether1".
+         Equivalent in RouterOS cli
+         "/ip address add address=1.1.1.1/32 interface=ether1"
+    type: str
+  remove:
+    description:
+      - Remove config/value from RouterOS by '.id'.
+         Example "*03" will remove config/value with "id=*03"
+         in selected path.
+          Equivalent in RouterOS cli "/ip address remove numbers=1"
+         ,note "number" in RouterOS cli is different from ".id"
```

Comment:
```suggestion
      - Remove config/value from RouterOS by '.id'.
      - Example C("*03") will remove config/value with "id=*03" in selected path.
      - Equivalent in RouterOS cli "/ip address remove numbers=1", note "number" in RouterOS cli is different from ".id".
```
Please take a look at https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html